reg/udral/README.md

```diff
@@ -236,6 +236,27 @@ It is not necessary to include the leading and trailing name component separator
 Registers that are not known to the auto-configuration authority are to be silently ignored,
 which permits addition of implementation-defined ports without breaking compatibility with standard services.
 
+
+#### Type signature register
+
+In many use cases it is desirable to have the ability to statically check the compatibility between sender and 
+receiver of a message. This capability could be used both during automatic configuration and by manual configurator tools.
+The design allows this by adding an extra register to every service pub/sub group that contains a string _message type signature_.
```

Comment:
```suggestion
The design allows this by adding an optional register per port that contains a string _data type signature_.
```

There are message types and service types. I expect your signature is applicable to both, is it not?

Reply:
Not yet. It should be easy to add (by adding a separator between request and reply part of the string and match both with AND condition. Tagged union implementation is not complete either.